lib/core/src/Cardano/Wallet.hs

```diff
@@ -446,10 +445,12 @@ data BlockchainParameters t = BlockchainParameters
         -- ^ Number of slots in a single epoch.
     , getTxMaxSize :: Quantity "byte" Word16
         -- ^ Maximum size of a transaction (soft or hard limit).
+    , getEpochStability :: Quantity "block" Word32
```

Comment:
"Epoch stability" seems a bit imprecise. It's the ouroboros security parameter _k_ here, isn't it? That has nothing to do with epochs, except that they currently happen to be _10×k_ slots in length. Could we call it "max unstable blocks" (the maximum number of blocks that could potentially be rolled back, a.k.a. "k") instead?

Reply:
The term ia borrowed from Jormungandr's terminology. Happy to give a more significant one though. I wonder however if "max unstable block" isn't too precise.